react/typings/events.d.ts

```diff
@@ -14,7 +15,7 @@ export interface PixelMessage extends MessageEvent {
     | SearchPageInfoData
     | UserData
     | CartIdData
-    | CartData
```

Comment:
There was no definition for the type `CartData`, so it was functioning as `any`, meaning that types weren't being enforced at all. 🫠

Removing it surfaced lots of type errors that were later fixed.